test/unity/toolkits/object_detection/test_od_serialization.cxx

```diff
@@ -62,7 +62,29 @@ BOOST_AUTO_TEST_CASE(test_init_darknet_yolo) {
                      "leakyrelu" + std::to_string(x.first) + "_fwd");
     TS_ASSERT_EQUALS(relulayer_.activation().leakyrelu().alpha(), 0.1f);
 
-    layer_num = layer_num + 3;
+    if (x.first <= 5){
+      const auto& poolinglayer_ = nn.layers(layer_num + 3);
```

Comment:
There seem to be a lot of magic numbers in this file. Could you add comments (or make variables) explaining what the magic numbers mean, so the code becomes more readable and inheritable?